### PR TITLE
ci: fix and enable some commands e2e specs

### PR DESCRIPTION
**Pull request by rerorero**
I just started looking into https://github.com/appium/appium/issues/16402
This PR aims for spec files directly under `test/functional/commands` dir. Subdirectories are not included.
I confirmed `npm run e2e-test:commands` passed on my laptop with `android-30;google_apis;arm64-v8a` emulator.

I have a question, what is `process.env.TESTOBJECT_E2E_TESTS` for? Do you still use it?

### CODE REVIEW

**Comment by KazuCocoa** [comment]
> process.env.TESTOBJECT_E2E_TESTS

It is maybe a legacy parameter recently we no longer use. TestObject is a service SauceLabs acquired long before.

**Comment by rerorero** [comment]
Got it, thank you for checking. Then I'll try to clean up them as well.

File: .github/workflows/functional-test.yml
Comment (rerorero) [inline .github/workflows/functional-test.yml:69]: Tried to have it [run in a different step ](https://github.com/appium/appium-uiautomator2-driver/pull/576/commits/d55d87e32747af34a7a00dfeee559c211ab2f31d) resulted in [a connection error to Appium server ](https://github.com/appium/appium-uiautomator2-driver/actions/runs/3968349318/jobs/6806217843)🤔 
Please let me know if you have better approaches to run multiple tests.

**Comment by rerorero** [comment]
@KazuCocoa Do you mind triggering the CI?

**Comment by KazuCocoa** [comment]
I've updated the actions' permission to require only a new github user. The next push won't require an approval

**Comment by rerorero** [comment]
That's very helpful thank you!!

**Comment by rerorero** [comment]
hum.. some tests seem to be flaky. Can you try `Re-run failed jobs`?

**Review by KazuCocoa — APPROVED** [review]
👍 
It's sufficient to pass a few devices. Others could be slow emulators etc. I'll merge after checking the test results

**Comment by KazuCocoa** [comment]
Hm, errors are rotation and attributes things. They might be related to emulator versions etc.
We can merge this right now since at least two versions have passed completely. Better than before

**Comment by KazuCocoa** [comment]
Thank you!

**Pull request merged**
